versions/development/SPEC.md

```diff
@@ -1990,8 +1926,8 @@ workflow myWorkflow {
 }
 ```
 
-#### Struct Assignment from Object Literal
-Structs can be assigned using an object literal. When Writing the object, all entries must conform or be coercible into the underlying type they are being assigned to
+#### Struct Assignment from Map Literal
+Structs can be assigned using an map literal. When Writing the literal, all entries must conform or be coercible into the underlying type they are being assigned to.
```

Comment:
This change implies that map literals do not need to have uniformly typed values.